### PR TITLE
[daint] Updating cray-python dependency of h5py

### PR DESCRIPTION
**Pull request by lucamar**
Changed dependency to `cray-python/3.6.1.1` to match the use of `Python 3` and address #610 .

### CODE REVIEW

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

File: easybuild/easyconfigs/p/PyExtensions/PyExtensions-3.6-CrayGNU-17.08.eb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pygelf/'],
+    }),
+    ('PyMySQL', '0.7.11', {
+        'req_py_majver': '3',
+        'req_py_minver': '6',
+        'source_urls': ['https://pypi.python.org/packages/source/p/pymysql/'],
+    }),
+    ('influxdb', '4.1.1', {
```
Comment (victorusu) [inline easybuild/easyconfigs/p/PyExtensions/PyExtensions-3.6-CrayGNU-17.08.eb]: I do not think we need influx anymore. 
@jgphpc, can you confirm?

Comment (jgphpc) [reply easybuild/easyconfigs/p/PyExtensions/PyExtensions-3.6-CrayGNU-17.08.eb]: yes, not needed anymore

Comment (lucamar) [reply easybuild/easyconfigs/p/PyExtensions/PyExtensions-3.6-CrayGNU-17.08.eb]: Thanks @jgphpc and @victorusu, I'll remove the package `influxdb` from the list of extensions

**Comment by lucamar** [comment]
`Dom` is under maintenance, therefore we will not be able to test the build there.

**Comment by lucamar** [comment]
retest this please

**Comment by lucamar** [comment]
retest this please

**Pull request merged**